cmd/operator/install.go

```diff
@@ -80,9 +80,18 @@ func NewCmdInstall() *cobra.Command {
 			}
 
 			kctl.SetArgs([]string{"apply", "-f", yaml})
-			//get original flags from kubectl
-
 			err = kctl.Execute()
+
+			if waitReady {
+				start := time.Now()
+				fmt.Printf("Waiting for core operator manager to be ready...\n")
+				err := k.WaitReady(context.Background(), namespace, "controller-manager")
```

Comment:
the name is calyptia-core-controller-manager. There should be a dynamic way to fetch deployment name

Reply:
have we got annotations/labels we can query by?

Reply:
yes we do
```
  labels:
    control-plane: controller-manager
    app.kubernetes.io/name: namespace
    app.kubernetes.io/instance: system
    app.kubernetes.io/component: manager
    app.kubernetes.io/created-by: operator
    app.kubernetes.io/part-of: operator
    app.kubernetes.io/managed-by: kustomize
```

Reply:
I mean those are terrible "Manager" is a bit generic

No "calyptia-core" or similar ones? What if we deploy other things like lua sandbox, etc.?

Reply:
We can adjust names any way we want

Reply:
we can change how it's done after we implement the other way to install the manifest